frontend/src/modules/dropdown-menu/data-hub.jsx

```diff
@@ -5,22 +5,16 @@ import { DownOutlined } from "@ant-design/icons";
 
 const DataHubDropdownMenu = withRouter(({ history }) => {
   return (
-    <Dropdown
-      overlayClassName="menu-dropdown-wrapper"
-      overlay={
-        <Menu className="menu-dropdown">
-          <Menu.Item className="nav-link">Data Map & Layers</Menu.Item>
-          <Menu.Item className="nav-link">Data Catalogue</Menu.Item>
-          <Menu.Item className="nav-link">Join Data Hub</Menu.Item>
-        </Menu>
-      }
-      trigger={["click"]}
-      placement="bottomRight"
+    <Button
+      type="link"
+      className="menu-btn nav-link menu-dropdown"
+      onClick={() => {
+        window.location.href = "http://datahub.gpmarinelitter.org/";
```

Comment:
I'm curious why this can't just be a link, instead of a button which sets `location.href`? That makes it impossible to open this link in a new tab, for instance. Why does this need to be a button?

Reply:
Hi @punchagan, thank you for the review. 
I will change that into `a href` link for both navbar & responsive menu.

Reply:
I don't know if the links should open in new tab, by default. In my previous comment, I was just referring to the user clicking on the link and opening in new tab. If we intend to pretend that the datahub is a part of the platform, I'm not sure if it should automatically open in a new tab.